### DISCUSSION TITLE
Is it possible to make a Chromebook/browser version?

### DISCUSSION THREAD

**Discussion by Chameleon-Coder** [obsproject/obs-studio#6342] [Development Questions] [answered]
Juat asking. I want to stream on Game Jolt, but I'm on a Chromebook without Linux features because administration blocked that.

**Comment by sebastian-s-beckmann** [comment, marked as answer]
No, it isn’t possible.
Even with the Linux features, ChromeOS doesn’t give us the tools required for recording the screen etc, so it’s completely unviable.
